src/main/java/net/fabricmc/loom/configuration/providers/forge/ForgeProvider.java

```diff
@@ -43,7 +43,15 @@ public ForgeProvider(Project project) {
 	@Override
 	public void provide(DependencyInfo dependency) throws Exception {
 		version = new ForgeVersion(dependency.getResolvedVersion());
-		addDependency(dependency.getDepString() + ":userdev", Constants.Configurations.FORGE_USERDEV);
+		String userdevClassifier;
+
+		if (Runtime.Version.parse(version.getMinecraftVersion()).compareTo(Runtime.Version.parse("1.13")) < 0) {
```

Comment:
Is this using the java internal version parser? If so, please don't

Reply:
`java.lang.Runtime.Version` appears to be public API since Java 9 as far as I can tell.

Reply:
It's intended for versions of the Java runtime and will break since Minecraft versions don't conform to that pattern. Please use something else.

Reply:
I made it manually compare the versions. Although it'll still fail on snapshot versions